website/src/app/pricing/page.tsx

```diff
@@ -0,0 +1,726 @@
+                  Remote access to your homelab
+                </span>
+              </li>
+              <li className="flex space-x-2.5">
+                <HiCheck className="flex-shrink-0 w-5 h-5 text-neutral-900" />
+                <span className="leading-tight text-neutral-900 ">
+                  Authenticate with magic link and OIDC
+                </span>
+              </li>
+              <li className="flex space-x-2.5">
+                <HiCheck className="flex-shrink-0 w-5 h-5 text-neutral-900" />
+                <span className="leading-tight text-neutral-900 ">
+                  Linux, macOS, iOS, ChromeOS, Android
+                </span>
+              </li>
+              <li className="flex space-x-2.5">
+                <HiCheck className="flex-shrink-0 w-5 h-5 text-neutral-900" />
+                <span className="leading-tight text-neutral-900 ">
+                  Community Support
+                </span>
+              </li>
+            </ul>
+          </div>
+          <div className="p-8 bg-neutral-50 border-2 border-neutral-200">
+            <h3 className="mb-4 text-2xl tracking-tight font-semibold text-primary-450">
+              Enterprise
+            </h3>
+            <p className="mb-8">
+              Zero trust network access for teams and organizations.
+            </p>
+            <h2 className="mb-16 text-2xl sm:text-4xl tracking-tight font-semibold text-neutral-900">
+              Contact us
+            </h2>
+            <div className="mb-16 w-full text-center">
+              <Link href="/contact/sales">
+                <button
+                  type="button"
+                  className="w-64 text-white font-bold tracking-tight rounded duration-0 hover:scale-105 transition transform shadow-lg text-lg px-5 py-2.5 bg-gradient-to-br from-accent-700 to-accent-600"
+                >
+                  Request a demo
+                </button>
+              </Link>
+            </div>
+            <p className="mb-2">
+              <strong>Everything in Starter, plus:</strong>
+            </p>
+            <ul role="list" className="font-medium space-y-2">
+              <li className="flex space-x-2.5">
+                <HiCheck className="flex-shrink-0 w-5 h-5 text-neutral-900" />
+                <span className="leading-tight text-neutral-900 ">
+                  SSO with Google Workspace
+                </span>
+              </li>
+              <li className="flex space-x-2.5">
+                <HiCheck className="flex-shrink-0 w-5 h-5 text-neutral-900" />
+                <span className="leading-tight text-neutral-900 ">
+                  Automatically sync users and groups
+                </span>
+              </li>
+              <li className="flex space-x-2.5">
+                <HiCheck className="flex-shrink-0 w-5 h-5 text-neutral-900" />
+                <span className="leading-tight text-neutral-900 ">
+                  Managed relay network
```

Comment:
Does this mean 'Free' tier users won't be able to use the Firezone Relays for data?

Reply:
Correct. They'll need to be behind a STUN-capable NAT. I was thinking this would be temporary until we can get some bandwidth limits in place on the Relays that depend on tier.